translations/messages.de.xlf

```diff
@@ -1770,6 +1770,10 @@
         <source>api_token_hidden</source>
         <target>Dies ist das einzige Mal, dass der Token angezeigt wird! Bewahren Sie ihn gut auf und vergewissern Sie sich, dass Sie ihn notiert haben, bevor Sie dieses Fenster schließen.</target>
       </trans-unit>
+      <trans-unit id="YXBpX3R" resname="api_token_qr_tooltip">
+        <source>api_token_qr_tooltip</source>
+        <target>Verwenden Sie diesen QR-Code, um den Arbeitsplatz in Ihrer Kimai Mobile App zu konfigurieren.</target>
```

Comment:
<target>Verwenden Sie diesen QR-Code, um den Arbeitsplatz in Ihrer Kimai Mobile App zu konfigurieren.</target>

Das ist absolut nicht generisch und passt nur auf Eure App. Wieso werden die offenen Feedbacks ignoriert und statt dessen wieder und wieder nach Merge gefragt?